Sources/Onboarding/Assets/Gradients.swift

```diff
@@ -0,0 +1,29 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import DesignSystem
+import UIKit
+
+enum OnboardingColor: String {
+  case background_gradient_step0 = "background-gradient-step0"
+  case background_gradient_step1 = "background-gradient-step1"
+  
+  var color: UIColor {
+    return UIColor(named: rawValue, in: .module, compatibleWith: nil)!
+  }
+}
+
+extension BraveGradient {
+  
+  public static var backgroundGradient: BraveGradient {
```

Comment:
Does this need to be `public`?

Reply:
No it is not you are right. Fix at https://github.com/brave/brave-ios/pull/6848/commits/1cd1100b84738c0c9c981283ebee2591f573a8ed